exercises/anagram/.meta/hints.md

```diff
@@ -0,0 +1,5 @@
+# Hints
+
+The solution is case insensitive, that means `"WOrd"` is the same with `"word"` or `"woRd"`. It may help to take a peak in the [std library](https://doc.rust-lang.org/std/index.html) for functions that can convert between them.
```

Comment:
Fixing some typos:

```suggestion
The solution is case insensitive, which means `"WOrd"` is the same as `"word"` or `"woRd"`. It may help to take a peek at the [std library](https://doc.rust-lang.org/std/index.html) for functions that can convert between them.
```

If you merge this, don't forget to regenerate README.md